Source/Private/ASDisplayNode+UIViewBridge.mm

```diff
@@ -514,15 +514,17 @@ - (BOOL)isOpaque
   return _getFromLayer(opaque);
 }
 
+
 - (void)setOpaque:(BOOL)newOpaque
 {
   _bridge_prologue_write;
-  
   BOOL shouldApply = ASDisplayNodeShouldApplyBridgedWriteToView(self);
   
   if (shouldApply) {
-    BOOL oldOpaque = _layer.opaque;
+    BOOL oldOpaque;
+    oldOpaque = _layer.opaque;
```

Comment:
Why not directly assigning: `BOOL oldOpaque = _layer.opaque;`

Reply:
The values of opaque differ depending on if it's a view or layer backed.